platform/ios/src/MGLMapView.mm

```diff
@@ -2305,14 +2326,9 @@ - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
             CGPoint west = [panGesture locationOfTouch:0 inView:panGesture.view];
             CGPoint east = [panGesture locationOfTouch:1 inView:panGesture.view];
```

Comment:
I realize you didn’t add these `west` and `east` in this PR, but perhaps we’d like to rename them here, too.